### DISCUSSION TITLE
Self reference many-to-many

### DISCUSSION THREAD

**Discussion by Gusterson** [strapi/strapi#6542] [Legacy Topics]
How do I create a many-to-many link for a content type that links to other items of the same type?

When I try to add a relationship field to, say, "Article" linking it many other related "Articles" when I try to use the "related" field on both sides I get a warning.

It doesn't seem right to have two columns in the admin (Related, and Related-other-end) to achieve this...

**Comment by Gusterson** [comment]
I see this: https://github.com/strapi/strapi/pull/5079 which says it closes the issue? But I don't see how to implement?

This comment: https://github.com/strapi/strapi/issues/3216#issuecomment-489695842

Seems to suggest it's possible with two different field-names - but this then pollutes my Admin with two different related fields. Confusing for admin user.

**Comment by Gusterson** [comment]
:cricket:

**Reply by Gusterson** [reply]
Hi @matheuschimelli since you went to the trouble to thumb down this, can you explain your solution to the issue if it's so obvious?

**Comment by Gusterson** [comment]
Can somebody point me in a direction?

**Reply by sandiprb** [reply]
@Gusterson  I think this is how it works in strapi. The possible solution is you can hide / disable one of the two fields in admin panel.
